psalm.xml

```diff
@@ -6,6 +6,8 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="psalm-baseline.xml"
+    ensureOverrideAttribute="false"
```

Comment:
This resulted in ~370 errors. I let you decide if you want to force this - I excluded it for this patch now since it would bloat this patch.